.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,134 @@
+    parser.add_argument(
+        "--files",
+        nargs="+",
+        help="List of changed files",
+        required=True,
+    )
+    args = parser.parse_args()
+
+    # Filter TypeScript files
+    ts_files = [f for f in args.files if f.endswith((".ts", ".tsx"))]
+    if not ts_files:
+        logging.info("No TypeScript files to check.")
```

Ruff (0.11.9)

`info()` call on root logger

(LOG015)